packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -852,6 +854,17 @@ class WorkItems:
     through the "RPA_OUTPUT_WORKITEM_PATH" env var a different path and name for this
     file.
 
+    **Mocking with Robocorp Code VSCode Extension**
```

Comment:
Mocking => Developing

Reply:
So I used "mock" in this instance as it is essentially a continuation and alternate way of performing was Ossi originally wrote just above it titled "Development and mocking" as this is the part were a robot developer needs to mock a work item so they can test their robot. Should we change Ossi's documentation as well to avoid the word mock entirely? I am not sure as that may lose the meaning of the doc section.

Reply:
No strong opinion here, but _Mocking_ in this scenario might be clearer since you're mocking a feature covered by the Cloud. (as in the same way you can use the CR Vault or the local _mocked_ Vault right from VSCode)

But at the same time that's not exactly mocking, it is actually local development and cloud environment simulation, so I'd go with Mika's suggestion if Ossi agrees as well to use the same term everywhere. (as I don't have a strong-opinion myself with this one)

Reply:
Based on slack discussion, switching all references of "mock" to "local development" or "cloud simulation".